cabal-install/cabal-install.cabal

```diff
@@ -320,6 +320,7 @@ library
         zlib       >= 0.5.3    && < 0.7,
         hackage-security >= 0.5.2.2 && < 0.6,
         text       >= 1.2.3    && < 1.3,
+        parsec     >= 3.1.13.0 && < 3.2,
```

Comment:
I see a note in this file:  `NOTE: when updating build-depends, don't forget to update version regexps in bootstrap.sh`.

Reply:
Also, is there a less-heavyweight library we could use, to decrease the size of the chicken-or-egg problem here? (Thx for this work, btw!)

Reply:
Cabal the library already uses parsec anyway.